### PR TITLE
Added PostgreSQL Type Casts Concept Entry

### LINKED ISSUE #4650: [Concept Entry] Postgresql type-casts
### Reason/inspiration (optional)

We would like a new entry on the `Type Casts` concept under PostgreSQL. The entry should go in a new file under `docs/content/postgresql/concepts/type-casts/type-casts.md`.
 
 The entry should include:
 
 - An introductory paragraph that describes the concept
 - A ‘Syntax’ section that includes the syntax for the concept (if applicable)
 - An ‘Example’ section that shows the concept in use
 
 Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by alexanderdarke**
### Description

Added a Concept Entry for PostgreSQL Type Casts - content/postgresql/concepts/type-casts/type-casts.md

### Issue Solved

Closes #4650

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/postgresql/concepts/type-casts/type-casts.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+```
+
+Output:
+
+```
+ int4
+------
+   38
+(1 row)
+```
```
Comment (cigar-galaxy82) [inline content/postgresql/concepts/type-casts/type-casts.md:46]:
```suggestion
| int4   |
| ------ |
| 38     |

File: content/postgresql/concepts/type-casts/type-casts.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+Output:
+
+```
+    date
+------------
+ 2024-05-22
+(1 row)
+```
```
Comment (cigar-galaxy82) [inline content/postgresql/concepts/type-casts/type-casts.md:61]:
```suggestion
| date     |
| -------- |
|2024-05-22| 
```

File: content/postgresql/concepts/type-casts/type-casts.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+Output:
+
+```
+ float8
+--------
+   22.5
+(1 row)
+```
```
Comment (cigar-galaxy82) [inline content/postgresql/concepts/type-casts/type-casts.md:76]:
```suggestion
|float8  |
| ------ |
|22.5    | 
```

**Comment by cigar-galaxy82** [comment]
Hey @alexanderdarke 

Thank you for contributing to Codecademy Docs!

I've suggested a few changes modify them at your convenience

Thank you!

**Comment by alexanderdarke** [comment]
Thanks @cigar-galaxy82! I've applied your suggestions and committed

**Comment by cigar-galaxy82** [comment]
Thank you @alexanderdarke

**Comment by letsbuildanything** [comment]
@cigar-galaxy82 your approving review is required here.

**Review by letsbuildanything — APPROVED** [review]
Ready to be merged! 🚀

**Pull request merged**